server/routes/viewModels/taskLists/caTasks.js

```diff
@@ -151,7 +152,9 @@ module.exports = {
         visible: curfewAddressApproved || bassChecksDone,
       },
       {
-        task: 'finalChecksTask',
+        title: 'Review case',
+        label: finalChecks.getLabel({ decisions, tasks }),
+        action: finalChecks.getCaProcessingAction({ decisions, tasks }),
```

Comment:
Now we have a consistent task design, use the standard task generator instead of the old custom one